File: 1752-check-if-array-is-sorted-and-rotated/1752-check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        vector<int> A=nums;
        sort(A.begin(),A.end());
        int n=nums.size();
        for(int x=0;x<n;x++)
        {
            bool is_match=true;
            for(int i=0;i<n;i++)
            {
                if(A[i]!=nums[(i+x)%n])
                {
                    is_match=false;
                    break;
                }
            }
            if(is_match)
            {
                return true;
            }
        }
        return false;
    }
};

File: 1752-check-if-array-is-sorted-and-rotated/1752-check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        A=sorted(nums)
        for x in range(len(nums)):
            is_match=True
            for i in range(len(nums)):
                if(A[i]!=nums[(i+x)%len(nums)]):
                    is_match=False
                    break
            if(is_match):
                return True
        return False
